src/games/home-button-test.js

```diff
@@ -0,0 +1,32 @@
+const { gameNode, Colors } = require("../common");
+
+class HomeButtonTest {
```

Comment:
this one can probably be removed, but I think I want this to be my test for "homegames common" that gives access to squisher/session, and maybe also force the home button to be rendered in a consistent place (eg. top corner). 

Something like a master root that contains homegames specific stuff and the developer gets a node under it, and that root renders a home button.